pages/storefront/edit.tsx

```diff
@@ -77,6 +79,10 @@ export default function Edit() {
     },
     { name: ['meta', 'title'], value: storefront?.meta.title ?? '' },
     { name: ['meta', 'description'], value: storefront?.meta.description ?? '' },
+    {
+      name: ['integrations', 'crossmintClientId'],
+      value: storefront?.integrations?.crossmintClientId ?? uuidv4(),
```

Comment:
Does the process fail if it doesn't have a value? Is the randomized value helpful in other regards? Just curious :)

Reply:
From crossmint perspective, we do need the client ID as it's what's used to enable the tool on our backend.

If it failed, we'd need a manual way to backfill it or else have the client create a new storefront, which feels veryy painful >.<